modules/ROOT/pages/tips-and-tricks.adoc

```diff
@@ -0,0 +1,119 @@
+= Tips and Tricks
+
+== Working with Toolbx
+
+=== Running applications from inside Toolbx on the host
+
+This can be necessary if you want to interact with tools available from the
+host, for example `podman`, `nmcli` or `rpm-ostree` without leaving the Toolbx
+container in between. You can use `flatpak-spawn`, included in the base
+installation for this:
+
+  $ flatpak-spawn --host podman --help
+
+If the application you want to call requires `sudo` access, the `-S` option must
+be supplied to `sudo` like below:
+
+  $ flatpak-spawn --host sudo -S rpm-ostree status
+
+If you find yourself using commands like these frequently to access e.g. the
+flatpak command from inside the Toolbx container, you can create yourself a
+short custom wrapper script (*inside the Toolbx container*):
+
+  $ echo '#!/bin/sh
+    exec /usr/bin/flatpak-spawn --host flatpak "$@"' | \
+    sudo tee /usr/bin/flatpak && sudo chmod +x /usr/bin/flatpak
+
+
+=== Finding out if you are currently in a Toolbx container
+
+If you frequently make use of Toolbx to perform various tasks and use multiple
+Toolbx containers it can be hard to keep track of whether you are currently
+executing commands on the host or in a Toolbx container. Furthermore, there is
+currently no command to tell you in which Toolbx container you are working.
+
+To alleviate this, you can add the following shell alias at the end of your
+`~/.bashrc`:
+
+  alias istoolbx='[ -f "/run/.toolboxenv" ] && grep -oP "(?<=name=\")[^\";]+" /run/.containerenv'
+
+When you open a new shell, you now have access to the new command `istoolbx`.
+This will behave as follows
+
+* When run from the host, returns an exit code of 1
+* When run from a Toolbx container, returns an exit code of 0 and prints the
+  current Toolbx containers name to the console
+
+[NOTE]
+====
+For this to become available inside your Toolbx containers, they must be
```

Comment:
For the alias to work? That's weird.

Reply:
Well spotted, that's wrong in this context. I previously posted this somewhere else where it was relevant, thanks!